build.gradle

```diff
@@ -186,15 +282,23 @@ dependencies {
 
   implementation group: 'org.hibernate', name: 'hibernate-core', version: '5.4.25.Final' // For CVE-2020-25638
   implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.13' // For CVE-2020-13956
+  implementation group: 'org.apache.activemq', name: 'activemq-client', version: '5.16.1' // For CVE-2021-26117
+  implementation group: 'org.apache.activemq', name: 'activemq-broker', version: '5.16.1' // For CVE-2021-26117
 
   compile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.14.0'
   compile group: 'org.apache.logging.log4j', name: 'log4j-to-slf4j', version: '2.14.0'
-  
+
   compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.40'
   compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: '9.0.40'
 
   runtime group: 'org.postgresql', name: 'postgresql', version: '42.2.18'
 
+  compile group: 'uk.gov.hmcts', name: 'befta-fw', version: '6.6.0-RDM-7165-2'
+
+  testCompile "org.junit.jupiter:junit-jupiter-api:${junitJupiterVersion}"
+  testRuntime "org.junit.jupiter:junit-jupiter-engine:${junitJupiterVersion}"
+  testRuntime "org.junit.vintage:junit-vintage-engine:${junitVintageVersion}"
```

Comment:
Could we just add these to the existing `libraries.junit5` array?

Reply:
done